src/networks.json

```
@@ -1551,6 +1551,21 @@
     "start": 8821493,
     "logo": "ipfs://QmNkr1UPcBYbvLp7d7Pk4eF3YCsHpaNkfusKZNtykL2EQC"
   },
+  "5000": {
+    "key": "5000",
+    "name": "Mantle",
+    "chainId": 5000,
+    "network": "mainnet",
+    "multicall": "0xcA11bde05977b3631167028862bE2a173976CA11",
+    "rpc": [
+      "https://rpc.mantle.xyz"
+    ],
+    "explorer": {
+      "url": "https://explorer.testnet.mantle.xyz"
```

This is not the testnet
```suggestion
      "url": "https://explorer.mantle.xyz"
```